Tests/SwiftFormatPrettyPrintTests/CommentTests.swift

```diff
@@ -416,4 +416,27 @@ public class CommentTests: PrettyPrintTestCase {
     
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 100)
   }
+
+  public func testCommentOnContinuationLine() {
+    let input =
+      """
+      func foo() {
+        return true
+          // comment
+          && false
+      }
+      """
+
+    let expected =
```

Comment:
`input` and `expected` are almost identical, do they need to be repeated?

Reply:
Updated to compare against `input + "\n"` instead (we do this in some other tests).